ext/src/Bigint.c

```diff
@@ -25,6 +25,15 @@
 #  define INT64_MIN (-INT64_MAX-1)
 #endif
 
+/* Ensure zend_long is defined for proper conversion */
+#ifndef zend_long
```

Comment:
What do you think about creating a new more descriptive type in `php_driver.h` for both PHP 5.6 and PHP 7 that conforms to the way we handle multiple PHP versions?

```
--- a/ext/php_driver.h
+++ b/ext/php_driver.h
@@ -105,6 +105,7 @@ typedef zval php5to7_zval;
 typedef zval *php5to7_zval_args;
 typedef zval *php5to7_zval_arg;
 typedef zend_string *php5to7_string;
+typedef zend_long php5to7_long;
 typedef zend_ulong php5to7_ulong;
 typedef zval php5to7_zend_resource_le;
 typedef zend_resource* php5to7_zend_resource;
@@ -268,6 +269,7 @@ php5to7_string_compare(php5to7_string s1, php5to7_string s2)
 typedef zval *php5to7_zval;
 typedef zval ***php5to7_zval_args;
 typedef char *php5to7_string;
+typedef long php5to7_long;
 typedef ulong php5to7_ulong;
 typedef zend_rsrc_list_entry php5to7_zend_resource_le;
 typedef zend_rsrc_list_entry *php5to7_zend_resource;
```